src/app/payment/paymentMethod/CheckoutcomCustomPaymentMethod.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from 'react';
+
+import { withLanguage, WithLanguageProps } from '../../locale';
+import { TextFieldForm } from '../creditCard';
+import { documentPaymentMethods, getDocumentOnlyValidationSchema } from '../documentOnly';
+
+import CreditCardPaymentMethod, { CreditCardPaymentMethodProps } from './CreditCardPaymentMethod';
+
+export interface CheckoutcomCustomPaymentMethodProps
+    extends Omit<CreditCardPaymentMethodProps, 'cardFieldset' | 'cardValidationSchema'> {
+    checkoutCustomMethod: string;
+}
+
+const checkoutcomCustomFieldset = (
```

Comment:
this should probably belong inside the method `CheckoutcomCustomPaymentMethod`, rather than outside